<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisassemblySettingsPage</class>
 <widget class="QWidget" name="DisassemblySettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Source Code Search Paths:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="KEditListWidget" name="sourcePaths">
     <property name="buttons">
      <set>KEditListWidget::Button::All</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Show branches:</string>
     </property>
     <property name="buddy">
      <cstring>showBranches</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="showBranches">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Show hexdump:</string>
     </property>
     <property name="buddy">
      <cstring>showHexdump</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="showHexdump">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tab width:</string>
     </property>
     <property name="buddy">
      <cstring>tabWidth</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="tabWidth"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="objdumpLabel">
     <property name="toolTip">
      <string>&lt;qt&gt;The &lt;tt&gt;objdump&lt;/tt&gt; binary that will be used for the disassembly view. Leave empty to let it be auto-detected.&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>Objdump:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditObjdump</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="lineEditObjdump">
     <property name="toolTip">
      <string>&lt;qt&gt;The &lt;tt&gt;objdump&lt;/tt&gt; binary that will be used for the disassembly view. Leave empty to let it be auto-detected.&lt;/qt&gt;</string>
     </property>
     <property name="placeholderText">
      <string>auto-detect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditObjdump</tabstop>
  <tabstop>showBranches</tabstop>
  <tabstop>showHexdump</tabstop>
  <tabstop>tabWidth</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
